src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj

```diff
@@ -2698,6 +2698,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
 				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks";
+				WARNING_CFLAGS = "-Wpartial-availability";
```

Comment:
Add useful warning flag into Xcode -> Make sure it warns properly if some APIs is absent in old OSX

Ref: https://stackoverflow.com/a/45075684/3127477